plugins/web-plugin/src/main/java/ru/sbtqa/tag/pagefactory/web/aspects/Highlight.java

```diff
@@ -0,0 +1,62 @@
+package ru.sbtqa.tag.pagefactory.web.aspects;
+
+import org.aeonbits.owner.ConfigFactory;
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Pointcut;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ru.sbtqa.tag.pagefactory.context.PageContext;
+import ru.sbtqa.tag.pagefactory.web.properties.WebConfiguration;
+import ru.sbtqa.tag.pagefactory.web.utils.WebExpectedConditionsUtils;
+
+@Aspect
+public class Highlight {
+
+    private static final WebConfiguration PROPERTIES = ConfigFactory.create(WebConfiguration.class);
+    private static final Logger LOG = LoggerFactory.getLogger(WebExpectedConditionsUtils.class);
+
+    @Pointcut("call(* org.openqa.selenium.WebElement.click()) && if()")
+    public static boolean isHighlighEnabled() {
+        return PROPERTIES.isHighlightEnabled();
+    }
+
+    @Around("isHighlighEnabled()")
+    public void highlight(ProceedingJoinPoint joinPoint) throws Throwable {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)